components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__measurements.xml

```diff
@@ -108,51 +399,268 @@
       </scope>
     </class>
     <property>
-      <content>{{include reference="PhenoTips.TabelarDataMacros" /}}
+      <content>{{include reference="PhenoTips.PatientSheetMacros" /}}
+
+{{include reference="PhenoTips.ChartServiceMacros" /}}
 
 {{velocity}}
-#set ($targetClassName = 'PhenoTips.MeasurementsClass')
-#if ($xcontext.action == 'edit')
-  $xwiki.ssx.use('XWiki.DateTimePicker', {'colorTheme' : "$xwiki.getSpacePreference('colorTheme')"})##
-  $xwiki.jsx.use('XWiki.DateTimePicker')##
-  #if ($config &amp;&amp; "$!{config.getProperty('enableImperialUnitSystem').value}" == '1')
-    #set ($measurementUnitInfo = "enable-unit-switch $!{config.getProperty('measurementUnitSystem').value}")
+$xwiki.jsx.use('XWiki.DateTimePicker')##
+$xwiki.ssx.use('PhenoTips.UIX_Field__measurements')##
+#set ($measurementClassId = $doc.getObjectNumbers('PhenoTips.MeasurementClass'))##
+#set ($objs = $services.query.xwql('from doc.object(PhenoTips.MeasurementsConfigurationClass) as obj').setLimit(1).execute())##
+#set ($itemDoc = $xwiki.getDocument($objs.get(0)))##
+#set ($configObj = $itemDoc.getObject('PhenoTips.MeasurementsConfigurationClass'))##
```

Comment:
I don't think this is very important, as the chances of it happening are tiny, but the user will get an ugly error if the configuration document is not found